cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -55,15 +55,15 @@ use frame_system::{
 use pallet_asset_conversion_tx_payment::AssetConversionAdapter;
 use pallet_nfts::{DestroyWitness, PalletFeatures};
 use pallet_xcm::EnsureXcm;
-pub use parachains_common as common;
 use parachains_common::{
 	impls::DealWithFees,
 	message_queue::*,
 	westend::{consensus::*, currency::*, fee::WeightToFee},
 	AccountId, AssetIdForTrustBackedAssets, AuraId, Balance, BlockNumber, Hash, Header, Nonce,
 	Signature, AVERAGE_ON_INITIALIZE_RATIO, DAYS, HOURS, MAXIMUM_BLOCK_WEIGHT,
-	NORMAL_DISPATCH_RATIO, SLOT_DURATION,
+	NORMAL_DISPATCH_RATIO,
 };
+pub use parachains_common::{westend::consensus::RELAY_CHAIN_SLOT_DURATION_MILLIS, SLOT_DURATION};
```

Comment:
Why are you re-exporting this? From what I can see it's fine to just use it without needing to be pub.

Changing it from re-exporting the full module to just these two consts is a breaking change anyway.

I recently removed these `pub use`s from the fellowship runtimes, maybe worth doing here too, as I don't see why anybody would add the whole runtime as a dependency just to get access to `parachains_common`.

Reply:
Looks like it was exported for use in the tests. I didn't notice that, IMO exports should not be interspersed with imports.

Reply:
Should I add back to the `pub use ... as common` to prevent the breaking change, or just remove this line?

Reply:
I think you could go with either option for that:

1. Removing enforces importing directly from common, where it is actually defined and is a breaking changed but if documented is completely legit IMO. The polkadot and kusama runtimes have just done this.

2. Leaving it in as is doesn't actually cost you anything and is the minimal diff for this PR, but continues with an unnecessary re-export

Up to you on that one, I was just highlighting that your previous changes were not purely internal to the runtime.

Reply:
I'm fixing this and it looks like a bunch of tests were using the re-exports. We will have to remember to change it back in https://github.com/paritytech/polkadot-sdk/pull/2949/ once the runtimes define their own SLOT_DURATION, so that the tests pick up the right constants. Not a big deal, worst case scenario if we forget to do that is some tests will fail.